### ISSUE TITLE
Cannot bootstrap a new application 

### ISSUE THREAD

**Issue by victorseara** [single-spa/create-single-spa#250]
Hi :) 

I have just generate a new root-config project and it's not initializing properly.

Here is my config options: 

```
? Directory for new project .
? Select type to generate single-spa root config
? Which package manager do you want to use? yarn
? Will this project use Typescript? Yes
? Would you like to use single-spa Layout Engine No
? Organization name (can use letters, numbers, dash or underscore) org
```

After bootstrap an application and run the start script I am receiving the following error on the browser console.
 
```
Uncaught (in promise) Error: Unable to resolve bare specifier '@org/root-config' from http://localhost:9000/ (SystemJS Error#8 https://git.io/JvFET#8)
    at resolve.js:12
    at c.O.resolve (resolve.js:8)
    at system-core.js:34
```

**Comment by filoxo** [comment]
This is likely due to the local import map not rendering because `isLocal` is being calculated as false. See this line: https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/src/root-config/templates/src/index.ejs#L44

You could try simply removing that conditional around that block and seeing if it works. Then you might need to dig into the webpack config and find why isLocal is being assigned to false when passed into the HtmlWebpackPlugin. It's possible that the newest version of webpack discards or changed how env values are passed in/read.

**Comment by victorseara** [comment]
Webpack was receiving `isLocal` as `boolean` and the condition passed to template it was considering it as a `string`. 

Just changed this in my `webpack.config` to get it fixed. 
```
isLocal: webpackConfigEnv && webpackConfigEnv.isLocal === true
```
Thanks for supporting me @filoxo.

**Comment by filoxo** [comment]
Very interesting. I might reopen this later, it would be good to investigate how and why this happens. Thanks for letting us know!

**Comment by joeldenning** [comment]
This is a regression bug with create-single-spa@2. Here's the change we made in create-single-spa@2:

```sh
# create-single-spa@1, with webpack 4
webpack-dev-server --mode=development --port 9000 --env.isLocal=true

# create-single-spa@2, with webpack 5
webpack serve --port 9000 --env isLocal
```

Notice that in webpack 4, we were doing `--env.isLocal=true` which converted it to a string, but that in webpack 5, we do `--env isLocal` which converts it to `true` as a boolean.

I'll submit a fix.

**Comment by joeldenning** [comment]
The regression was introduced as part of https://github.com/single-spa/create-single-spa/pull/226.

**Comment by joeldenning** [comment]
Fixed in #251

**Comment by joeldenning** [comment]
Fix released in https://github.com/single-spa/create-single-spa/releases/tag/v2.0.2
